build/Targets/Settings.props

```diff
@@ -11,6 +11,8 @@
     <DeveloperBuild Condition="'$(OfficialBuild)' != 'true' AND '$(JenkinsBuild)' != 'true'">true</DeveloperBuild>
     <RepoRoot>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)..\..\'))</RepoRoot>
     <NuGetToolPath>$(RepoRoot)nuget.exe</NuGetToolPath>
+    <!-- Disable the implicit nuget fallback folder as it makes it hard to locate and copy ref assemblies to the test output folder -->
```

Comment:
@agocke If we remove the step for copying netstandard ref assemblies to test output folders, we could re-enable fallback folder. You may want to take a note in the issue you file. Thanks